api/build.gradle

```diff
@@ -91,6 +89,11 @@ dependencies {
 
     compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: '0.0.4'
     compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version:'0.0.4'
+    //Added this for spring boot upgrade as swagger was not working with the latest version.
```

Comment:
@AnilDhawaleGitHub - Is your application not starting up without these change?

Reply:
yes its failing with building the application only u can refer build no 60 for this pr